python/dgl/backend/pytorch/sparse_emb.py

```diff
@@ -0,0 +1,160 @@
+import torch as th
+from .tensor import attach_grad, is_recording
+from .util import get_shared_mem_array, create_shared_mem_array
+from datetime import timedelta
+
+from ... import ndarray as nd
+
+_store = None
+
+class NodeEmbedding: # NodeEmbedding
+    '''Sparse node embeddings for graph.
+
+    DGL provides a sparse embedding to support models that require learnable embeddings.
```

Comment:
Maybe something like this. I think it's better to avoid using "sparse embedding". I'm not sure if "sparse embedding" is a well-known term.

"When training node embeddings on a graph, the embedding layer is typically very large.
We need to use train the embedding layer with sparse gradient updates in order to scale to
a graph with many nodes. `NodeEmbedding` is optimized for such use case. It provides
an efficient way to only update the embeddings involved in a mini-batch and can scale to
a graph with millions of nodes. It also supports multi-GPU training efficiently by partitioning
the embeddings."